### PR TITLE
add performance metrics collecting script

### PR DESCRIPTION
**Pull request by jimyang2008**

### CODE REVIEW

File: scripts/perf_metrics.sh [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+    # output
+    case $OUTPUT_FORMAT in
+
+        $OUTPUT_SHELL)
+            for gw in $gws; do
+                ping -nc 6 $gw | awk '/rtt/ {print $4}'| {
+                    IFS=/ read min avg max mdev
+                    gw_ip=$(echo $gw|tr '.' '_')
+                    cat <<EOL
+ping_${gw_ip}_min  $min
```
Comment (MelvinTo) [inline scripts/perf_metrics.sh:219]: there should be only one gw in the environment, it can also reduce the length of name

Comment (jimyang2008) [reply scripts/perf_metrics.sh:219]: Will take the first one if multiple gateways do exist.

**Comment by MelvinTo** [comment]
eth0_rx
eth0_tx
ping_min
ping_avg
ping_max
ping_mdev
temp
cpufreq_0
cpufreq_1
cpufreq_2
cpufreq_3
cpu_usr
cpu_nice
cpu_iowait
cpu_soft
cpu_idle
p_zeek
p_openvpn
p_firemain
p_firemon

**Comment by jimyang2008** [comment]
> eth0_rx
> eth0_tx
> ping_min
> ping_avg
> ping_max
> ping_mdev
> temp
> cpufreq_0
> cpufreq_1
> cpufreq_2
> cpufreq_3
> cpu_usr
> cpu_nice
> cpu_iowait
> cpu_soft
> cpu_idle
> p_zeek
> p_openvpn
> p_firemain
> p_firemon

Changed as suggested

**Pull request merged**